src/samplers/const_sampler.js

```diff
@@ -19,11 +19,18 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import * as constants from '../constants.js';
+
 export default class ConstSampler {
     _decision: boolean;
+    _tags: Array<Tag>;
 
     constructor(decision: boolean) {
         this._decision = decision;
+        this._tags = [
+            {'key': constants.SAMPLER_TYPE_TAG_KEY, 'value': constants.SAMPLER_TYPE_CONST},
+            {'key': constants.SAMPLER_PARAM_TAG_KEY, 'value': this._decision.toString()}
```

Comment:
you do not need `toString()` it, that will be done by the span if needed, and in the future won't be needed at all because we can preserve the type of the tag explicitly in jaeger model.